Automattic/wp-calypso
# Implement Help-Center contact form

First: Terribly sorry for the big  PR. It's just the irreducible complexity of dependencies pulling each other. The majority of the changes (in file count, I mean), are one-liners to update the import path.

#### Changes proposed in this Pull Request

* This PR introduces a new Calypso-independent site-picker component. This picker is keyboard navigable and works on mobiles. The Calypso version is insanely entangled with Calypso state, and I couldn't take it out without breaking things. And it pulled 100KB worth of deps with it.
* It also moves `Gravatar`, `HappinessEngineersTray`, and `useHappinessEngineersQuery`, `safeImageUrl`,  to packages. 
* Implements the Help-Center contact form with three modes `CHAT`, `EMAIL`, and `FORUM`.

#### Testing instructions
- Checkout the branch
- yarn dev --sync from apps/editing-toolkit
- Go to https://YOUR_SITE. wordpress.com/wp-admin/post-new.php?enable-help-center
- Go to this file https://github.com/Automattic/wp-calypso/blob/add/help-center-contact-form/apps/editing-toolkit/editing-toolkit-plugin/help-center/src/contents.js#L70 and change the `mode` prop from CHAT, EMAIL, and FORUM. All modes should be supported by the form.
- Smoke test Calypso (My home, help/contact, etc..).
